app/server/express.js

```diff
@@ -137,6 +138,8 @@ app.use("/api/login", initLogger, loginRouter, requestFinally);
 app.use("/api/logout", initLogger, logoutRouter, requestFinally);
 app.use("/api/token", initLogger, tokenRouter, requestFinally);
 
+app.use("/api/search", initLogger, searchRouter, requestFinally);
```

Comment:
I may be overlooking something, but what's the reasoning for not calling authenticateToken on this endpoint?

Reply:
Thanks, that should have been there.